packages/office-ui-fabric-react/src/components/HoverCard/examples/HoverCard.EventListenerTarget.Example.tsx

```diff
@@ -0,0 +1,80 @@
+import * as React from 'react';
+import { HoverCard, IPlainCardProps, HoverCardType } from 'office-ui-fabric-react/lib/HoverCard';
+import { DirectionalHint } from 'office-ui-fabric-react/lib/common/DirectionalHint';
+import { Fabric } from 'office-ui-fabric-react/lib/Fabric';
+import { mergeStyleSets } from 'office-ui-fabric-react/lib/Styling';
+import { IconButton } from 'office-ui-fabric-react';
+
+const classNames = mergeStyleSets({
+  plainCard: {
+    width: 200,
+    height: 200,
+    display: 'flex',
+    alignItems: 'center',
+    justifyContent: 'center'
+  },
+  target: {
```

Comment:
I don’t see you using this className.

Reply:
in function `_onRenderPlainCard`

Reply:
I mean I don’t see `classNames.target` assigned anywhere

Reply:
right, removed